apps/system-e2e/src/tests/judicial-system/smoke/custody-tests.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test } from '@playwright/test'
+import * as prosecutor from './custody-prosecutor'
+import * as court from './custody-court'
+
+test.describe.serial('Custody tests', () => {
+  prosecutor.addTests()
+  court.addTests()
+})
```

Comment:
If the tests _need_ to be run after each other, maybe put them in the same file and section? If not, why test them serially?

Reply:
I did it like this because there are still around 4 more tests (with 2 new user roles) that need to be added to complete the chain, and I was worried a single file would become monstrous and really hard to work with 🥲